core/src/main/java/com/linecorp/armeria/client/endpoint/healthcheck/DefaultHealthCheckerContext.java

```diff
@@ -291,4 +292,12 @@ CompletableFuture<?> release() {
         }
         return null;
     }
+
+    @Override
+    public String toString() {
+        return MoreObjects.toStringHelper(this)
+                          .add("originalEndpoint", originalEndpoint)
+                          .add("endpoint", endpoint)
```

Comment:
The following properties seem useful for debugging.
```suggestion
                          .add("endpoint", endpoint)
                          .add("initialized", handle != null)
                          .add("destroyed", destroyed)
                          .add("refCnt", refCnt)
```

Reply:
I'm wondering if whether we received the first check could also be useful.

```suggestion
                          .add("initializationStarted", handle != null)
                          .add("initialized", initialCheckFuture.isDone())
```